config/locales/fr.yml

```diff
@@ -150,24 +154,24 @@ fr:
       web: Web
     action_logs:
       actions:
-        assigned_to_self_report: "%{name} s'est assigné le signalement de %{target} à eux-même"
-        change_email_user: "%{name} a modifié l'adresse de courriel de l'utilisateur %{target}"
-        confirm_user: "%{name} adresse courriel confirmée de l'utilisateur %{target}"
-        create_custom_emoji: "%{name} a importé de nouveaux emoji %{target}"
+        assigned_to_self_report: "%{name} s’est assigné le signalement de %{target} à eux-même"
+        change_email_user: "%{name} a modifié l’adresse de courriel de l’utilisateur·rice %{target}"
+        confirm_user: "%{name} adresse courriel confirmée de l’utilisateur·ice %{target}"
+        create_custom_emoji: "%{name} a importé de nouveaux émoji %{target}"
         create_domain_block: "%{name} a bloqué le domaine %{target}"
         create_email_domain_block: "%{name} a mis le domaine du courriel %{target} sur liste noire"
-        demote_user: "%{name} a rétrogradé l'utilisateur %{target}"
+        demote_user: "%{name} a rétrogradé l’utilisateur·ice %{target}"
         destroy_domain_block: "%{name} a débloqué le domaine %{target}"
         destroy_email_domain_block: "%{name} a mis le domaine du courriel %{target} sur liste blanche"
         destroy_status: "%{name} a enlevé le statut de %{target}"
-        disable_2fa_user: "%{name} a désactivé l'authentification à deux facteurs pour l'utilisateur %{target}"
-        disable_custom_emoji: "%{name} a désactivé l'emoji %{target}"
-        disable_user: "%{name} a désactivé le login pour l'utilisateur %{target}"
-        enable_custom_emoji: "%{name} a activé l'emoji %{target}"
-        enable_user: "%{name} a activé le login pour l'utilisateur %{target}"
+        disable_2fa_user: "%{name} a désactivé l’authentification à deux facteurs pour l’utilisateur·ice %{target}"
+        disable_custom_emoji: "%{name} a désactivé l’émoji %{target}"
+        disable_user: "%{name} a désactivé le login pour l’utilisateur·ice %{target}"
+        enable_custom_emoji: "%{name} a activé l’émoji %{target}"
+        enable_user: "%{name} a activé le login pour l’utilisateur·ice %{target}"
         memorialize_account: "%{name} a transformé le compte de %{target} en une page de mémorial"
-        promote_user: "%{name} a promu l'utilisateur %{target}"
-        remove_avatar_user: "%{name} a supprimé l'avatar de %{target}'s"
+        promote_user: "%{name} a promu l’utilisateur·ice %{target}"
+        remove_avatar_user: "%{name} a supprimé l’avatar de %{target}"
         reopen_report: "%{name} a ré-ouvert le signalement %{target}"
```

Comment:
réouvert or rouvert — rouvrir is by far the most frequent and ré-ouvrir is much more rare.